test/eternity.jl

```diff
@@ -35,7 +35,7 @@ end
 
     ncolors = maximum(puzzle[:,2:end])
 
-    m = Model(CSJuMPTestSolver())
```

Comment:
the semantic of the `CSJuMPTestSolver()` doesn't fit. change to e.g. `CSJuMPTestOptimizer()` :)